content/millennium/r4/clinical/diagnostics/questionnaire-response.md

```diff
@@ -4,14 +4,13 @@ title: QuestionnaireResponse | R4 API
 
 # QuestionnaireResponse
 
-<%= beta_tag %>
-
 * TOC
 {:toc}
 
 ## Overview
 
 The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. A given QuestionnaireResponse must belong to a corresponding Questionnaire. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The resource should be leveraged as a snapshot in time and new data should be consistently retrieved through the API rather than stored within an application.
```

Comment:
I dont even know if this is true, i just know the item.id might change! and the id-modifiable extension might change too